src/librbd/object_map/SnapshotRemoveRequest.cc

```diff
@@ -22,17 +22,11 @@ void SnapshotRemoveRequest::send() {
   assert(m_image_ctx.snap_lock.is_wlocked());
 
   if ((m_image_ctx.features & RBD_FEATURE_FAST_DIFF) != 0) {
-    compute_next_snap_id();
-
-    uint64_t flags;
-    int r = m_image_ctx.get_flags(m_snap_id, &flags);
-    assert(r == 0);
+    int r = m_image_ctx.get_flags(m_snap_id, &m_flags);
+    ceph_assert(r == 0);
```

Comment:
@dillaman Was it your intention to use the new style assert here? Because in other commits it looked like you preferred to use the old stile assert.

Reply:
@trociny updated